project/src/main/java/edu/depaul/cdm/se452/project/model/ProfessorRepository.java

```diff
@@ -0,0 +1,15 @@
+package edu.depaul.cdm.se452.project.model;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+//repository class to help perform CRUD operations on the professors table
+public interface CourseRepository extends CrudRepository<Student, Long> {
+	//find a professor by passing a course number
+	List<Professor> findProfByCourse(int num);

```

Comment:
This seemed to cause run-time issues for right now as it is trying to look for a Course property in the Professor class.  Can probably remove this for the time being and maybe add back in after we figure out the SQL.